### PR TITLE
Use aqp::checkHzDepthLogic for horizon depth logic checks

### PR DESCRIPTION
**Pull request by brownag**

### CODE REVIEW

File: R/simplifyArtifactData.R [hunk 1]
```diff
@@ -6,75 +6,75 @@
...
-  
+
+
   ## split flat/nonflat
   idx <- grep('^flat', x$huartshp, ignore.case = TRUE, invert=TRUE)
   arts <- x[idx, ]
-  
+
   idx <- grep('^flat', x$huartshp, ignore.case = TRUE)
   farts <- x[idx, ]
```
Comment (dylanbeaudette) [inline R/simplifyArtifactData.R:31]: busted

**Review by dylanbeaudette — APPROVED** [review]
Thanks for doing this. Do you want have or want before / after sample data set for comparisons? I figured that this is part of your process but happy to send something over.

File: R/simplifyArtifactData.R (hunk 1 above)
Comment (brownag) [reply R/simplifyArtifactData.R:31]: ?

**Comment by brownag** [comment]
I am in the process of testing the more unusual methods. All NASIS things work without error, per the routine I have for testing all the different query methods. It is not convenient for me to work with your before/after datasets, but feel free to let me know if any of the following methods are not working as expected:

**Comment by brownag** [comment]
Tested and seems to be working as expected for `fetchNASIS('pedon_report')` and `fetchPedonPC`/`getHzErrorsPedonPC` PedonPC .accdb containing new CA750 pedons from @jskovlin

File: R/simplifyArtifactData.R (hunk 1 above)
Comment (dylanbeaudette) [reply R/simplifyArtifactData.R:31]: a joke, the variable name caught my eye and my inner 8 year old liked it.

**Comment by dylanbeaudette** [comment]
> I am in the process of testing the more unusual methods. All NASIS things work without error, per the routine I have for testing all the different query methods. It is not convenient for me to work with your before/after datasets, but feel free to let me know if any of the following methods are not working as expected: fetchNASIS_report, fetchPedonPC, getHzErrorsPedonPC

I have no good ways to test those three functions, sorry.

Is the before / after testing part of this process? If not then I am offering some assistance. If it is, then feel free to ignore.

**Pull request merged**